R/miniconda.R

```diff
@@ -0,0 +1,197 @@
+
+#' Install Miniconda
+#'
+#' Download the [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
+#' installer, and use it to install Miniconda.
+#'
+#' @param path The path in which Miniconda will be installed. Note that the
+#'   installer does not support paths containing spaces.
+#' 
+#' @param version The version of Python associated with the Miniconda installer
```

Comment:
I think this can only strictly be `2` or `3`. Users may think that they can specify something like `3.6` or `3.7`, in which case the installer URL won't be valid. It might be good to make that clear in the documentation and validate that it is only `2` or `3` in the code.